backend/app/configs/docker/kafka/bitnami_kafka.yaml

```
@@ -14,9 +14,12 @@ services:
       KAFKA_CFG_PROCESS_ROLES: broker,controller
       KAFKA_CFG_CONTROLLER_LISTENER_NAMES: CONTROLLER
 
-      KAFKA_CFG_LISTENERS: INTERNAL://kafka1:29092,CONTROLLER://kafka1:2181,EXTERNAL://0.0.0.0:9092
-      KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP: CONTROLLER:PLAINTEXT,INTERNAL:PLAINTEXT,EXTERNAL:PLAINTEXT
-      KAFKA_CFG_ADVERTISED_LISTENERS: INTERNAL://kafka1:29092,EXTERNAL://localhost:9092
+      KAFKA_CFG_LISTENERS: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)